src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,127 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Class to encapsulate Duke Chatbot.
+ */
+public class Duke {
+
+    private final Ui dukeUi;
+    private final Parser dukeParser;
+    private final Storage dukeStorage;
+    private EntryList entries;
+
+    private final String TERMINATION_COMMAND = "bye";
```

Comment:
I like how the constants are named using uppercase and underscores, makes the styling very organized.